### ISSUE TITLE
Cannot call peeked method with [ref] parameters

### ISSUE THREAD

**Issue by vors** [oising/poke#1]
First of all, this is an excellent module, thank you!

I cannot call method with ref / out parameters.
Consider (this API is public, I use it just as an example)

``` powershell
$tokens = $null
$errors = $null
$ast = [System.Management.Automation.Language.Parser]::ParseInput("foo", [ref]$tokens, [ref]$errors)
```

And now with peek:

``` powershell
$tokens = $null
$errors = $null
$parser = [System.Management.Automation.Language.Parser] | peek
$ast = $parser.ParseInput("foo", [ref]$tokens, [ref]$errors)
```

![image](https://cloud.githubusercontent.com/assets/816680/6091282/33157f40-ae59-11e4-8d14-3c59b58564d4.png)

**Comment by oising** [comment]
Thanks for the report. I'll take a crack at this on the plane ride home on Friday.

**Comment by oising** [comment]
didn't manage last flight, but did this time (currently posting from a delta flight out of houston ;)) fixed for 1.1 -- you can clone the repo now to get the enhancement.
